frontend/src/pages/Common/Budget.js

```diff
@@ -93,7 +93,7 @@ export default class Budget extends React.Component {
         <Table>
           <TableHead>
             <TableRow>
-              <TableCell>{strings.common.organization}</TableCell>
+              <TableCell>{strings.users.organization}</TableCell>
```

Comment:
This should be in the common section of the translation since it its a "common" component

Reply:
Will be done in #286